code/workspaces/infrastructure-api/src/kubernetes/serviceAccountApi.js

```diff
@@ -1,11 +1,29 @@
 import logger from '../config/logger';
 import { getCoreV1Api } from './kubeConfig';
 
+async function namespacedServiceAccountExists(name, namespace) {
+  const k8sApi = getCoreV1Api();
+  logger.info(`Checking serviceAccount ${name} exists in ${namespace}`);
```

Comment:
Might be worth making this a `debug` log? The important information looks like it is covered in the logs of the functions using this one and if there are too many then it might clutter them up a bit.